.gitmodules

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: 2006-2024 Knut Reinert & Freie Universität Berlin
+# SPDX-FileCopyrightText: 2016-2024 Knut Reinert & MPI für molekulare Genetik
+# SPDX-License-Identifier: CC0-1.0
```

Comment:
Does this need a new line? The submodules aren't checked out in the ci

Reply:
Yeah, this doesn't seem to work like this at all :/ added a .gitmodules.license

Reply:
Looks like some commit removed the `submodules/*` directories. I tried readding them